Application/Source/Food Table/FoodTableViewModel.swift

```diff
@@ -0,0 +1,35 @@
+import RxSwift
+import RxCocoa
+import RxExtensions
+import Presentations
+import Action
+
+class FoodTableViewModel: ViewModel {
```

Comment:
Nit: I'd prefer not to have `Table` in the name of the `ViewModel` (or `ViewController` unless there is also a `Collection` version as well).

Should this instead be called this `IngredientsViewModel`? `FoodListViewModel`? `FoodViewModel`?